src/plugins_3rdparty/primer3/src/primer3_core/oligotm.c

```diff
@@ -83,7 +86,7 @@ All rights reserved.
  */
 
 /* Table 1 (old parameters):
- * See table 2 in the paper [Breslauer KJ, Frank R, Bl�cker H and
+ * See table 2 in the paper [Breslauer KJ, Frank R, Bl�cker H and
```

Comment:
It was better

Reply:
Looks like the original sources were written with some non-standard encoding and now after the change my editor fails to convert them correctly to UTF-8. 
I fixed the bad char manually.